app/javascript/utilities/getFocusTrapToggle.js

```diff
@@ -0,0 +1,32 @@
+import { createFocusTrap } from 'focus-trap';
+
+export function getFocusTrapToggle(containerSelector) {
```

Comment:
documentation (blocking): Exported functions require JS Doc documentation. For an example, see https://github.com/forem/forem/blob/master/app/javascript/utilities/http/request.js#L1-L23